src/main/java/domain/Rank.java

```diff
@@ -0,0 +1,34 @@
+package domain;
+
+public enum Rank {
+    FIRST(6, 2_000_000_000, false),
+    SECOND(5, 30_000_000, true),
+    THIRD(5, 1_500_000, false),
+    FOURTH(4, 50_000, false),
+    FIFTH(3, 5_000, false),
+
+    NONE(0, 0, false);
```

Comment:
(질문) 이 부분은 당첨과 낙첨을 구분하기위해 줄바꿈을 해주신걸까요? 😃

```suggestion
    NONE(0, 0, false);
```

Reply:
네!! 당첨 부분과 낙첨 부분을 분리하는 것이 더 가독성이 좋을 것이라고 생각했습니다!